Sources/EdgeAgentUpdater/EdgeAgent.swift

```diff
@@ -0,0 +1,66 @@
+import ArgumentParser
+import EdgeAgentGRPC
+import Foundation
+import GRPCCore
+import GRPCNIOTransportHTTP2
+import Logging
+import _NIOFileSystem
+
+@main
+struct EdgeAgentUpdater: AsyncParsableCommand {
+    static let configuration = CommandConfiguration(
+        commandName: "edge-agent-updater",
+        abstract: "Edge Agent Updater"
+    )
+
+    @Option(name: .shortAndLong, help: "The path to the Edge Agent binary.")
+    var agentPath: String = "/usr/local/bin/edge-agent"
```

Comment:
port and path will need to be configurable

Reply:
They are already